filebeat/module/kibana/_meta/fields.yml

```diff
@@ -5,5 +5,41 @@
   fields:
     - name: kibana
       type: group
-      description: >
+      description: ''
```

Comment:
I think it would be useful to have a description here, as it impacts the documentation as well.

Example:
```
- name: audit
  type: group
  release: beta
  description: >
    Module for parsing Snyk audit logs.
  fields:
```

Reply:
Done in [`e35d4d6` (#22696)](https://github.com/elastic/beats/pull/22696/commits/e35d4d68638a05ace19fb905b65538d72e2d502e)